sonar-db/src/main/java/org/sonar/db/property/PropertiesDao.java

```diff
@@ -170,6 +172,10 @@ public PropertyDto selectProjectProperty(DbSession dbSession, long componentId,
     return executeLargeInputs(keys, partitionKeys -> getMapper(session).selectByKeys(partitionKeys, componentId));
   }
 
+  public List<PropertyDto> selectGlobalPropertiesByKeyQuery(DbSession session, String keyQuery) {
+    return getMapper(session).selectGlobalPropertiesByKeyQuery(buildLikeValue(keyQuery, WildcardPosition.BEFORE));
```

Comment:
Indeed, it was for perf reason, but as it's not explicit I will use a BEFORE_AND_AFTER position